hack/docs/internal/cronjob-tutorial/controller-implementation.go

```diff
@@ -0,0 +1,528 @@
+package cronjob
+
+
+const ControllerIntro = `
```

Comment:
Do we need to export this ones? `ControllerIntro`

Could we have like `controllerCommentFragment `

Reply:
Do you recommend I put those fragments in `generate_cronjob.go`?

Currently I split tutorial contents(mainly words) of each book page and put them in different files. Wish this can help easier maintenance in the future.

If we want to keep them in the same file, I can also move them back.